docs/rancher-vmware.md

```diff
@@ -7,183 +7,79 @@ title: ''
+    ...
 ```
 
-Remove the existing content and enter this code snippet and click on save. Remember to modify this as per your need.
-
-:::info main options
-name: elemental-cls1 --> "Change this as per your need"
-
-device: /dev/sda --> Make sure your target device is "sda" otherwise find out and change the naming convention here, for example in raspberry pi it could be "mmblk---"
-
-emulate-tpm: true  --> Use this only if your target device doesnt have a tpm device and you have a way of emulating tpm like in VMware or KVM
+`emulated-tpm-seed: -1` is sets the client to use a random seed to compute TPM hash, this is useful to be capable
+to reuse the same registration end point definition for multiple machines. See further [TPM documentation](tpm.md).
 
-emulated-tpm-seed: 1 --> increase this by 1 for every new machine. --> If this is not given each machine will recieve the same tpm hash so at any point in time regardless of how many machines you boot it wont show up under the machine inventory.
-:::
+## Step 2: Create the installation ISO
 
-:::danger attention
-Emulate TPM is only for non-production usage like for testing as it beats the purpose of security. So in production usage use the code above without the emulate-tpm and emulate-tpm-seed
-:::
+The installation media needs to be tied to an specific registration end point. This is created and handled
+with the [SeedImage](seedimage-reference.md) resource.
 
-Once you create the machine registration end point it should show up as active.
+Consider follwing example:
 
-![Machine registered in Registration Endpoints](images/rancher-vmware-machine-registered.png)
+<CodeBlock language="yaml" title="seedimage.yaml" showLineNumbers>{SeedImage}</CodeBlock>
 
-Next click on it to view the registration url and download the initial-registration yaml.
+Once the SeedImage resources is created it starts building an ISO with the provided OS image and linking it to
+the given registration end point. Once done a download URL will be available in the SeedImage resource status.
 
-![Initial registration URL](images/rancher-vmware-initial-registration-url.png)
+You can download it with:
 
-Hit the download button, this will download the "initial-registration.yaml" file on to your local system.
-
-The registration yaml looks something like this:
-
-![Example of initial registration YAML](images/rancher-vmware-initial-registration-url-yaml.png)
-
-We will use this file and inject its content into our vanilla elemental iso.
-
-## Part 3: Create the ISO
-
-Make sure you have initial-registration.yaml in the system where you will create the iso in this we will use a Linux vm.
-
-Create a directory to keep everything:
-
-```shell showLineNumbers
-mkdir /home/tux/elemental-demo && cd /home/tux/elemental-demo
+```shell
+wget --no-check-certificate $(kubectl get machineregistration -n fleet-default fire-nodes -o jsonpath="{.status.registrationURL}") -O initial-registration.yaml
```

Comment:
Here we need the link to the SeedImage ISO link, right?

```suggestion
wget --no-check-certificate --content-disposition $(kubectl get seedimages.elemental.cattle.io -n fleet-default fire-img -o jsonpath="{.status.downloadURL}")
```

Reply:
Indeed I copied the command for the quick start guide and completely forgot to adapt it to this specific use case 😭